### PR TITLE
S2t2 index helper bootstrap

### PR DESCRIPTION
**Pull request by matthewsearle01**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?php
+require "vendor/autoload.php";
+
+use Transformers\viewhelpers\IndexViewHelper;
+
+$indexViewHelper = new IndexViewHelper();
+// $indexViewHelper->searchTransformers('Gr');
```
Comment (BelindaWilliams) [inline index.php]: Delete this line - part of story 4

File: src/services/Search.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
```
Comment (BelindaWilliams) [inline src/services/Search.php]: Remove this file - part of Story4

File: src/viewhelpers/IndexViewHelper.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    public function createTransformersList(): string
+    {
+        $str = "";
+        foreach ($this->transformerList as $item) {
+            $str .= $this->createTransformerCard($item);
+        }
+        return $str;
+    }
+
+    public function searchTransformers(string $search): void
```
Comment (BelindaWilliams) [inline src/viewhelpers/IndexViewHelper.php]: Delete this function - part of Story4

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+a:hover {
+  color: #c20f02;
+}
+
+footer a {
+  color: #ffffff;
+  position: fixed;
+  bottom: 20px;
```
Comment (BelindaWilliams) [inline css/styles.css:60]: Edit pixel sizing? Should this be responsive?

File: index.php (hunk 1 above)
Comment (matthewsearle01) [reply index.php]: Done

File: src/services/Search.php (hunk 2 above)
Comment (matthewsearle01) [reply src/services/Search.php]: Done

**Pull request merged**